mock/py/mockbuild/util.py

```diff
@@ -1043,7 +1043,8 @@ def set_config_opts_per_cmdline(config_opts, options, args):
     config_opts.update(cli_opt_new)
 
     config_opts['verbose'] = options.verbose
-    config_opts['print_main_output'] = config_opts['verbose'] > 0 and sys.stderr.isatty()
+    if 'print_main_output' not in config_opts or config_opts['print_main_output'] is None:
```

Comment:
This will not work if someone set `print_main_output` to False as suggested in `site-defaults.cfg`. Can you rather make it `... or not config_opts['print_main_output']`? And the default in `site-defaults.cfg` set `False`? Because the line below set it to a boolean value.

Reply:
What specifically would not work if `print_main_output` were set to False? It should be that `True` will make it always print, `False` will make it never print, and `None` (or not set) will make it do the tty check.

If you feel that my explanation was insufficient in the `site-defaults.cfg`, I can explicitly say that a value of `None` or leaving it unset will use the tty check.

Reply:
Ahh, you meant it this way... Yes explanation in `site-defaults.cfg` that "None will use tty check, False means disabled, True means enabled" would be helpful.

Reply:
Added language in site-defaults.cfg that explicitly says `None` or leaving it defined will use the default (tty-check) behavior.